images/win/scripts/Installers/Install-Chrome.ps1

```diff
@@ -3,24 +3,40 @@
 ##  Desc:  Install Google Chrome
 ################################################################################
 
-Import-Module -Name ImageHelpers -Force
+Import-Module -Name ImageHelpers -Force;
 
-$temp_install_dir = 'C:\Windows\Installer'
-New-Item -Path $temp_install_dir -ItemType Directory -Force
+$ChromeInstallerFile = "chrome_installer.exe";
+$ChromeInstallerUri = "https://dl.google.com/chrome/install/375.126/${ChromeInstallerFile}";
+Install-Exe -Url $ChromeInstallerUri -Name $ChromeInstallerFile -ArgumentList ("/silent", "/install")
 
-Install-MSI -MsiUrl "https://seleniumwebdrivers.blob.core.windows.net/knownchromeversion/googlechromestandaloneenterprise64.msi" -MsiName "googlechromestandaloneenterprise64.msi"
+Write-Host "Adding the firewall rule for Google update blocking";
+New-NetFirewallRule -DisplayName "BlockGoogleUpdate" -Direction Outbound -Action Block -Program "C:\Program Files (x86)\Google\Update\GoogleUpdate.exe";
```

Comment:
One of the asks we got from customers was to allow updating the browser on demand on the fly if needed by customers.

This need generally comes in if we are late in making an update here after a new browser version has been released. Can we validate if updating chrome via chocolatey or some other means works after these changes?